pages/installation.rst

```diff
@@ -33,6 +33,8 @@ This chapter is explaining the many ways to install Graylog and aims to help cho
 System requirements
 ===================
 
+.. warning:: We recommend that you avoid Elasticsearch 7.11; it will cause your instance to break. 
```

Comment:
Can we make this a little clearer and say that you absolutely shouldn't because it *will* break?

Is this the only place in the docs where we mention Elasticsearch versions?